pages/what-we-do/overview.md

```diff
@@ -21,28 +21,37 @@ tags:
 </div>
 
 <div class="content-block">
-  <div class="container container--sm vertical-padding">
+  <div class="container container--xs vertical-padding">
 
 The National Leadership Centre was launched by the Chancellor in 2018 as a three year pilot, following the recommendations of [the Public Services Leadership Taskforce](https://www.gov.uk/government/publications/national-leadership-centre). The Taskforce was established in 2017 to advise on the scope for a new body to support public service leaders.
 
-<div class="header-highlight">
+<div class="pdf-left-block">
+	<div class="pdf-left-block__pdf">
+		<a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/799567/6.4846_CO_CPSL-Report_A4-P_WEB_NoLogo__002.pdf">
+		<img src="../../static/img/taskforce-report-pdf.png" alt="Image of taskforce report"/>
+		<span>October 2018: Better Public Services Report by the Public Services Leadership Taskforce</span>
+		</a>
+		<span>PDF, 1010KB, 52 pages</span>
```

Comment:
Could we move this text into the link? It then gets read out by screen readers as part of the link

Reply:
Moved 👍